Link_sinFade_Library.ino

```diff
@@ -260,62 +263,63 @@ void HSVehtoRGB(float HSVfColor[], float rgbf_color[]) {
   float Sat = HSVfColor[1];  // 0 to 100.
   float Val = HSVfColor[2];  // 0 to 100.
 
-  int rangeNumber = HSVehRange(HSVfColor);      ////////  Uses: HSVehRange(hsvColorF).  <<<< Can I do away with "rangeNumber" ?
+  // rgb's are in 0.0 ... 1.0
```

Comment:
It might be hard mentally to make the switch from 0..255 to 0.0 ... 1.0.
I suppose I could switch back to _float_ 0 .. 255, and take that as the canonical pwm range.